src/js/footer/helper/index.js

```diff
@@ -161,7 +161,13 @@ const getDomainAppID = () => {
 };
 
 export const loginUrl = () => {
-  const server_url = localStorage.getItem("config.server_url");
+  const server_url = () => {
+    try {
+      return localStorage.getItem("config.server_url") || "green.derivws.com";
+    } catch (error) {
+      return "green.derivws.com";
```

Comment:
I don't suggest hardcoding the green url in multiple places, since it is also being used in the websocket file. 
You can reference it from a single place.